spa/src/components/account/ForgotPassword/index.js

```diff
@@ -0,0 +1,80 @@
+import { useState, useReducer } from 'react';
+import { useForm } from 'react-hook-form';
+
+// AJAX
+import axios from 'ajax/axios';
+import { FORGOT_PASSWORD_ENDPOINT } from 'ajax/endpoints';
+
+import * as S from '../Account.styled';
+
+import ForgotPasswordForm from './ForgotPasswordForm';
+
+import Logobar from 'components/account/common/logobar/Logobar';
+import Leftbar from 'components/account/common/leftbar/Leftbar';
+
+import { SIGN_IN } from 'routes';
+import YellowSVG from 'assets/images/account/yellow-bar.svg';
+
+// State management
+import fetchReducer, { initialState, FETCH_START, FETCH_SUCCESS, FETCH_FAILURE } from 'state/fetch-reducer';
+
+// Analytics
+import { useConfigureAnalytics } from 'components/analytics';
+
+function ForgotPassword() {
+  useConfigureAnalytics();
+  const [forgotPasswordState, dispatch] = useReducer(fetchReducer, initialState);
+  const formSubmitErrors = forgotPasswordState?.errors?.detail;
+
+  const [infoMessage, setInfoMessage] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const onForgotPasswordSubmit = async (fdata) => {
+    dispatch({ type: FETCH_START });
+    try {
+      await axios.post(FORGOT_PASSWORD_ENDPOINT, { email: fdata.email });
+      setInfoMessage('Success. If your email is registered, an email with a reset-link will be sent. ');
```

Comment:
Would be good to use a constant so that we can test if the text is being displayed correctly
```suggestion
      setInfoMessage('Success. If your email is registered, an email with a reset-link will be sent.');
```

Reply:
done